src/components/common/header/header.scss

```diff
@@ -163,7 +163,26 @@ header {
     }
 
     
-  }
+  } 
+
+  
+}  
+
+.popupdisplayHIDE{
+  display: none
+} 
+
+.popupdisplaySHOW{
+  display: flex; 
```

Comment:
I think we should start a document on css convensions for our team. For now I think kabob case is good.

Reply:
noted. Just to confirm you would like kebob case for css classes? This would translate to the className .popup-display-show in the jsx. (I will be using the already provided .hide in the future. )

Reply:
Yes. That would probably be the easiest to remember for now until we decide on something more complicated.

You might not even need a "show" class if just removing the "hide" class can work.